tests/acceptance/requirements.txt

```diff
@@ -0,0 +1,3 @@
+PyHamcrest>=1.8.3
+git+https://github.com/qaenablers/python-qautils.git@develop#egg=python-qautils
+python-keystoneclient==1.3.0
```

Comment:
this requirement is already defined in the global file of the project. Is it necessary to add this dependency again?